stl/src/vector_algorithms.cpp

```diff
@@ -1114,12 +1112,7 @@ namespace {
         }
 
         static uint64_t _Get_any_u(const __m128i _Cur) noexcept {
-#ifdef _M_IX86
-            return (static_cast<uint64_t>(static_cast<uint32_t>(_mm_extract_epi32(_Cur, 1))) << 32)
-                 | static_cast<uint64_t>(static_cast<uint32_t>(_mm_cvtsi128_si32(_Cur)));
-#else // ^^^ x86 / x64 vvv
-            return static_cast<uint64_t>(_mm_cvtsi128_si64(_Cur));
-#endif // ^^^ x64 ^^^
+            return _Minmax_traits_8::_Get_any(_Cur);

```

Comment:
`_Minmax_traits_8::_Get_any` returns `_Minmax_traits_8::_Signed_t` so this is a value-modifying conversion to `uint64_t`.

However, if we "punch through" and call `_Minmax_traits_8::_Get_v_pos`, that returns `_Unsigned_t` so no `static_cast` would be needed.